### PR TITLE
[DOC] Add optimizer group docs

### PR DESCRIPTION
**Pull request by hameizi**
## Why are the changes needed?
Add optimizer group docs

## Brief change log

Add optimizer group docs

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: site/docs/en/guides/managing-optimizers.md [hunk 1]
```diff
@@ -10,84 +10,119 @@ The type of Local Container is local, which is a way to start Optimizer by local
...
+in the "export.{env_arg}" property of the container's properties. The commonly used configurations are as follows:
 
-- FLINK_HOME, download the Flink installation package and unzip it. Take Flink-1.12.7 as an example, download https://archive.apache.org/dist/flink/flink-1.12.7/flink-1.12.7-bin-scala_2.12.tgz , assuming that it is extracted to /opt/ directory, then configure the value /opt/ flink-1.12.7/
-- Since the Flink distribution does not come with the hadoop compatible package flink-shaded-hadoop-2-uber-x.y.z.jar, you need to download it and copy it to the FLINK_HOME/lib directory. The flink-shaded-hadoop-2-uber-2.7.5-10.0.jar is generally sufficient and can be downloaded at: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop- 2-uber/2.7.5-10.0/flink-shaded-hadoop-2-uber-2.7.5-10.0.jar
+- flink-home, download the Flink installation package and unzip it. Take Flink-1.12.7 as an example, 
+download https://archive.apache.org/dist/flink/flink-1.12.7/flink-1.12.7-bin-scala_2.12.tgz , assuming that it is 
+extracted to /opt/ directory, then configure the value /opt/ flink-1.12.7/. Since the Flink distribution does not come
```
Comment (Aireed) [inline site/docs/en/guides/managing-optimizers.md:26]: remove the space here 'opt/ fllink-1.12.7/'

Comment (Aireed) [inline site/docs/en/guides/managing-optimizers.md:26]: Should we enclose paths in backticks?  @zhoujinsong  what do you think ?

File: site/docs/en/guides/deployment.md [hunk 2]
```diff
@@ -248,24 +242,9 @@ containers:
       export.HADOOP_CONF_DIR: "/etc/hadoop/conf/"                   # Hadoop config dir
       export.HADOOP_USER_NAME: "hadoop"                             # Hadoop user submit on yarn
       export.FLINK_CONF_DIR: "/etc/hadoop/conf/"                    # Flink config dir
```
Comment (Aireed) [inline site/docs/en/guides/deployment.md:244]: should we add  memory configuration here ?

File: site/docs/en/guides/managing-optimizers.md [hunk 3]
```diff
@@ -10,84 +10,119 @@ The type of Local Container is local, which is a way to start Optimizer by local
...
+| ARCTIC_HOME                      | Arctic  root directory                                         |
 | -a AMS_THRIFT_SERVER_URL         | The address of the AMS thrift service, for example: thrift://10.196.98.23:18112, can be obtained from the config.yaml configuration. |
-| -qn OPTIMIZE_GROUP_NAME          | The name of the optimize group is configured in config.yaml and can also be found in the front-end Optimizer-->Optimizer group, the container in this group must be of external type. |
-| -p EXECUTOR_PARALLELISM          | Flink Task Parallelism                                             |
+| -g OPTIMIZE_GROUP_NAME           | The name of the optimize group is configured in config.yaml and can also be found in the front-end Optimizer-->Optimizer group, the container in this group must be of external type. |
+| -p EXECUTOR_PARALLELISM          | Flink Task Parallelism                                      |
 | -m EXECUTOR_MEMORY               | Execution memory, the sum of JM's memory and TM's memory for the Flink task, is used to report to AMS to count the optimizer's resource usage. |
-| --heart-beat               | optimizer heartbeat reporting interval  |
```
Comment (Aireed) [inline site/docs/en/guides/managing-optimizers.md:63]: lost  parameter  `-hb`

**Review by Aireed — COMMENTED** [review]
i have left some comments, PTAL

File: site/docs/en/guides/deployment.md (hunk 2 above)
Comment (hameizi) [reply site/docs/en/guides/deployment.md:244]: The "memory" information is configured within the optimizer group.

**Pull request closed without merging**